package.json

```diff
@@ -22,9 +22,10 @@
     "gen:nft": "openapi-generator-cli generate -i apps/nft-service/src/api/swagger.json -g typescript-axios -o apis/nft/ts -c apis/nft/client-ts-spec.json",
     "genapis": "lerna run gen:swagger && cross-env JAVA_OPTS=\"-Dlog.level=warn\" run-p gen:*",
     "build:src": "lerna run build",
-    "build:all": "run-s clean genapis build:src",
-    "rebuild": "cross-env CONFIG_NAME=prod yarn build:all",
-    "build": "cross-env CONFIG_NAME=prod yarn build:src",
+    "build:storybook": "build-storybook",
+    "build:all": "run-s clean genapis build:src build:storybook",
+    "build": "cross-env CONFIG_NAME=prod yarn build:all",
+    "rebuild": "cross-env CONFIG_NAME=prod yarn build:src",
```

Comment:
NOTE: Changing build command behaviour.

Previously, `rebuild` was the complete rebuild, and `build` was incremental.  However, this is non-standard, and occasionally would result in errors when we used the incremental build when we needed full build.

Reply:
So we've reverted back to having the default `build` command clean & build everything, and `rebuild` is the one we can use if we explicitly want an incremental build